applications/DEMApplication/custom_constitutive/DEM_rolling_friction_model_viscous_torque.cpp

```diff
@@ -0,0 +1,106 @@
+/////////////////////////////////////////////////
+// Main author: Rafael Rangel (CIMNE)
+// Email: rrangel@cimne.upc.edu
+// Date: Oct 2022
+//
+// Description:
+// This model can be found as Model Type B in [Jun Ai, 2011, Assessment of rolling resistance models in discrete element simulations]
```

Comment:
IDK in DEMApp but this header is not standard :)

Reply:
Changed the headers